<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTractographyDisplayModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerTractographyDisplayModuleWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="FiberBundleTableDisplay">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Simple Display</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <property name="collapsedHeight">
      <number>0</number>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_table">
      <item row="1" column="0">
       <widget class="qMRMLTractographyDisplayTreeView" name="TractographyDisplayTreeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="listenNodeModifiedEvent" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="ctkCheckBox" name="SolidTubeColorCheckbox">
        <property name="text">
         <string>Solid Tube Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="percentageWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="percentageOfFibersShownLabel">
        <property name="text">
         <string>Percentage of Fibers Shown</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkSliderWidget" name="percentageOfFibersShown">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="FiberBundleEditing">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Fiber Bundle Selection</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <property name="collapsedHeight">
      <number>0</number>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="qSlicerTractographyEditorROIWidget" name="ROIEditorWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Advanced Display</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <property name="collapsedHeight">
      <number>0</number>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="TractDisplayModesTabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="LineTab">
         <attribute name="title">
          <string>Line</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="qSlicerTractographyDisplayWidget" name="LineDisplayWidget" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_13">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="TubeTab">
         <attribute name="title">
          <string>Tube</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="qSlicerTractographyDisplayWidget" name="TubeDisplayWidget" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_15">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="GlyphTab">
         <attribute name="title">
          <string>Glyph</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="qSlicerTractographyDisplayWidget" name="GlyphDisplayWidget" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="qSlicerTractographyDisplayGlyphWidget" name="GlyphPropertiesWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_16">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_14">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerTractographyDisplayWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerTractographyDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerTractographyDisplayGlyphWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerTractographyDisplayGlyphWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerTractographyEditorROIWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerTractographyEditorROIWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setFiberBundleNode(vtkMRMLNode*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qMRMLTractographyDisplayTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLTractographyDisplayTreeView.h</header>
   <slots>
    <signal>currentNodeChanged(vtkMRMLNode*)</signal>
    <slot>setCurrentNode(vtkMRMLNode*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerTractographyDisplayModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>GlyphDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyDisplayModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TubeDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyDisplayModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>LineDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyDisplayModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>GlyphPropertiesWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyDisplayModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TractographyDisplayTreeView</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyDisplayModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ROIEditorWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TractographyDisplayTreeView</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>qSlicerTractographyDisplayModuleWidget</receiver>
   <slot>setFiberBundleNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TractographyDisplayTreeView</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>ROIEditorWidget</receiver>
   <slot>setFiberBundleNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>currentNodeChanged(vtkMRMLNode*)</signal>
  <slot>setFiberBundleNode(vtkMRMLNode*)</slot>
 </slots>
</ui>
